src/main/webapp/WEB-INF/views/orders/order-details.jsp

```diff
@@ -0,0 +1,22 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Order details</title>
+</head>
+<body>
+<h1>Your order details</h1>
+<table border="1">
+    <tr>
+        <th>Order ID</th>
+        <th>User ID</th>
+        <th>Products</th>
```

Comment:
this is your only JSP to show order details (no matter if you came here from "/user/orders" or "/orders"

I meant it will be better to show product info here **in the table**, and not orderId and userId